presto-druid/src/test/resources/common.runtime.properties

```diff
@@ -1,4 +1,4 @@
-druid.extensions.loadList=["druid-hdfs-storage", "druid-kafka-indexing-service", "druid-datasketches"]
+druid.extensions.loadList=["druid-hdfs-storage", "druid-kafka-indexing-service", "druid-datasketches", "druid-stats"]
```

Comment:
What happens if the extension is not loaded?

Reply:
I think druid just throws an exception saying the aggregate function the user is asking for is not supported by druid.

Reply:
Is the extension typically used, or rarely?
this implies the default behavior (should agg pushdown be enabled by default or not)

Reply:
In my experience this is typical because in absence of it druid can't support any stddev or variance function. I would say  it is rare that it is not in the installation given the utility it provides vs the friction or overhead of having it in an installation. 

We could disable pushdown by default or we can make a rest call (no sql way to get list of loaded extensions from druid as of now, I can propose to them to add  this) to get list of extension and only pushdown stats functions if the extension is loaded.